vtex-apps/quickorder
# [Enhancement] Changing the validate button in TextArea and Upload blocks to only be shown if the user has written something or sent a file, respectively

#### What does this PR do? \*

Changes the validate button in Upload and TextArea blocks to appear only if the user has written something or sent a file.

#### How to test it? \*

Open Quick order and check the text area and upload components before and after adding data to it.

#### Describe alternatives you've considered, if any. \*

None

#### Related to / Depends on \*

None
